packages/hardhat-ignition-core/src/internal/execution/jsonrpc-client.ts

```diff
@@ -687,7 +688,8 @@ export class EIP1193JsonRpcClient implements JsonRpcClient {
 
       // Logic copied from ethers v6
       const maxFeePerGas =
-        latestBlock.baseFeePerGas * 2n + maxPriorityFeePerGas;
+        (this._config?.maxFeePerGas ?? latestBlock.baseFeePerGas) * 2n +
+        maxPriorityFeePerGas;
```

Comment:
This doesn't look correct to me, conceptually. I think it should be something like:

```typescript
      const maxFeePerGas =
        this._config?.maxFeePerGas ?? ((latestBlock.baseFeePerGas) * 2n +
        maxPriorityFeePerGas);
```

Otherwise the `maxFeePerGas` in the config is being interpreted as the base fee of the block, which doesn't make sense to me.

Reply:
Ah! You're absolutely right